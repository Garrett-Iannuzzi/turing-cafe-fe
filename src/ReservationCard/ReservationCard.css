article {
  border: solid 3px black;
  background-color: white;
  padding: 10px;
  margin: 2px;
}