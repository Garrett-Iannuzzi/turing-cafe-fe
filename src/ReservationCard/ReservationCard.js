import React from 'react';
import './ReservationCard.css';

const ReservationCard = ({ id, name, date, time, number }) => {
  return(
    <article>
      <h1 className='article-h1'>Name: {name}</h1>
      <h2 className='article-h2'>Date: {date}</h2>
      <h3 className='article-h3'>Time: {time}</h3>
      <h3 className='article-h4'>Number of guests: {number}</h3>
    </article>
  )
}

export default ReservationCard;