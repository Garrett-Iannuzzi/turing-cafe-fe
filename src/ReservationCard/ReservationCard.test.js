import React from 'react';
import { shallow } from 'enzyme';
import ReservationCard from '../ReservationCard/ReservationCard';

describe('ReservationCard', () => {

  it('Should match the snapshot with the correct data', () => {
    const wrapper = shallow(<ReservationCard />)

    expect(wrapper).toMatchSnapshot()
  });
});