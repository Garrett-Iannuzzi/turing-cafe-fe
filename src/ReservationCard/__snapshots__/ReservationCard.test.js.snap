// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should match the snapshot with the correct data 1`] = `
<main
  className="app"
>
  <h1
    className="app-title"
  >
    Turing Cafe!
  </h1>
  <Form
    handleAddReservation={[Function]}
  />
  <ReservationContainer
    reservations={Array []}
  />
</main>
`;

exports[`ReservationCard Should match the snapshot with the correct data 1`] = `
<article>
  <h1
    className="article-h1"
  >
    Name: 
  </h1>
  <h2
    className="article-h2"
  >
    Date: 
  </h2>
  <h3
    className="article-h3"
  >
    Time: 
  </h3>
  <h3
    className="article-h4"
  >
    Number of guests: 
  </h3>
</article>
`;
